### DISCUSSION TITLE
Muting Categories

### DISCUSSION THREAD

**Discussion by catuhana** [stoatchat/discussions#520] [Feature Suggestions]
Why muting every single channel in a category while I can just mute the category.. please?

**Comment by cotto-n** [comment]
Makes sense, it would help you just easily mute channels at once.

**Comment by celestialdruid** [comment]
I use the browser web app and it pains me having to individually mute all the channels I want muted every single time I open a new browser. (I clear cookies and history on close.) Please implement this.
